usdSchemas/GafferCycles.usda

```diff
@@ -0,0 +1,130 @@
+#usda 1.0
+(
+	subLayers = [
+		@usdLux/schema.usda@,
+		@usd/schema.usda@
+	]
+)
+
+over "GLOBAL" (
+	customData = {
+		string libraryName = "GafferCycles"
+		bool skipCodeGeneration = 1
+		bool useLiteralIdentifier = 1
+	}
+)
+{
+}
+
+# Here we define a bunch of codeless auto-apply API schemas for extending the
+# standard UsdLux lights with inputs specific to Cycles. This approach is
+# modelled on the one used by UsdRiPxr to add RenderMan-specific inputs, and
+# we believe is the one Pixar intends everyone to use.
+
+class "GafferCyclesLightAPI" (
+	customData = {
+		token[] apiSchemaAutoApplyTo = [ "DistantLight", "DiskLight", "DomeLight", "RectLight", "SphereLight" ]
+		string apiSchemaType = "singleApply"
+		string className = "GafferCyclesLightAPI"
+	}
+	inherits = </APISchemaBase>
+)
+{
+
+	string inputs:cycles:lightgroup = "" (
+		displayGroup = "Basic"
+		displayName = "Light Group (Cycles)"
+	)
+
+	bool inputs:cycles:use_mis = true (
+		displayGroup = "Refine"
+		displayName = "Multiple Importance (Cycles)"
+	)
+
+	bool inputs:cycles:use_camera = true (
+		displayGroup = "Refine"
+		displayName = "Visible to Camera (Cycles)"
+	)
```

Comment:
I tested this with a RectLight in Gaffer's viewer, and it didn't seem to have any effect? Is there some plumbing we need in the Cycles backend to pass the values through?

Reply:
Oh silly me I forgot the important part in ShaderNetworkAlgo that takes away the namespace prefix, fixed and should work now.